src/js/structs/Application.js

```diff
@@ -7,57 +9,28 @@ import TaskStats from './TaskStats';
 import VolumeList from './VolumeList';
 
 module.exports = class Application extends Service {
-  getAcceptedResourceRoles() {
-    return this.get('acceptedResourceRoles');
-  }
 
-  getArguments() {
-    return this.get('args');
-  }
-
-  getCommand() {
-    return this.get('cmd');
-  }
-
-  getContainerSettings() {
-    return this.get('container');
-  }
-
-  getCpus() {
-    return this.get('cpus');
-  }
-
-  getContainer() {
-    return this.get('container');
-  }
-
-  getConstraints() {
-    return this.get('constraints');
-  }
+  // getContainer() {
```

Comment:
Blast. That should not be there. I commented that part out until I figure out if it was used or not. Apparently we are not using this function anywhere. Instead we are using the `getContainerSettings` (in spec).